src/cppext.cpp

```diff
@@ -11,7 +11,10 @@
 
 #include "cdd/kernel.h"
 
-#define ADBM(NAME) raw_t* NAME = allocDBM(size)
+#include <dbm/fed.h>
+#include <dbm/print.h>
+
+#define ADBM(NAME) raw_t* NAME = allocDBM(cdd_clocknum)
```

Comment:
```suggestion
#define ADBM(NAME, DIM) raw_t* NAME = allocDBM(DIM);
```
In C++ we should avoid using macros: they are very pesky and prone to errors. 
We need to rewrite this library to have better interfaces...